code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1864,6 +1926,13 @@
 	color = "#FAEAFF"
 	taste_description = "synthetic catnip"
 
+/datum/reagent/moonsugar/on_mob_life(mob/living/carbon/M)
+	if(prob(20))
+		to_chat(M, "You find yourself unable to supress the desire to meow!")
+		M.emote("nya")
```

Comment:
say("*nya") not emote emote will result in "Person Name nya" rather than "Person Name lets out a nya!"

Reply:
... at least I think? tell me if I'm wrong.

Reply:
This was the first bit of code I ever wrote for fermichem. It's a little nostalgic.